Soccar [Game-Server]/src/nl/soccar/gameserver/SessionWrapper.java

```diff
@@ -0,0 +1,120 @@
+package nl.soccar.gameserver;
+
+import nl.soccar.gameserver.message.JoinSessionMessage;
+import nl.soccar.gameserver.message.PlayerJoinedSessionMessage;
+import nl.soccar.gameserver.message.PlayerLeftSessionMessage;
+import nl.soccar.gameserver.message.PlayerStartedGameMessage;
+import nl.soccar.library.*;
+import nl.soccar.library.enumeration.TeamColour;
+import nl.soccar.socnet.Node;
+import nl.soccar.socnet.connection.Connection;
+
+public final class SessionWrapper {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)